### ISSUE TITLE
KafkaHealthCheck needs to be configurable

### ISSUE THREAD

**Issue by cer** [eventuate-foundation/eventuate-cdc#60]
https://github.com/eventuate-foundation/eventuate-cdc/blob/441f5214f7c7718f9cc3975c2c1d05d5e1e3e150/eventuate-local-java-cdc-connector-unified/src/main/java/io/eventuate/local/unified/cdc/pipeline/common/health/KafkaHealthCheck.java#L22

**Comment by cer** [comment]
from user:

```
It is properties like sasl jaas config, security protocol (to change to SASL_SSL), ssl endpoint identification algorithm (to change to https) and sasl mechanism. The ideal would be to have it configurable the same way through ENV/Spring variable as currently done for producers/consumers
```

**Comment by silarr** [comment]
To help and make it clear, here is an example (full names) of parameters for consumer and producer passed through Env variables:
  EVENTUATE_LOCAL_KAFKA_CONSUMER_PROPERTIES_SASL_MECHANISM: PLAIN
  EVENTUATE_LOCAL_KAFKA_CONSUMER_PROPERTIES_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: https
  EVENTUATE_LOCAL_KAFKA_CONSUMER_PROPERTIES_SASL_JAAS_CONFIG: org.apache.kafka.common.security.plain.PlainLoginModule required username="Q4TARSDCREBSDSDGRE" password="Fscjer00sBDLwbrPSOPSmQdKLT0psSDTSDfRI6pgrAxfABD09aQzDdre2SDrfesd+Ug";
  EVENTUATE_LOCAL_KAFKA_CONSUMER_PROPERTIES_SECURITY_PROTOCOL: SASL_SSL
  EVENTUATE_LOCAL_KAFKA_CONSUMER_PROPERTIES_BOOTSTRAP_SERVERS: pkc-6ksd5.us-east1.gcp.confluent.cloud:9092

  EVENTUATE_LOCAL_KAFKA_PRODUCER_PROPERTIES_SASL_MECHANISM: PLAIN
  EVENTUATE_LOCAL_KAFKA_PRODUCER_PROPERTIES_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: https
  EVENTUATE_LOCAL_KAFKA_PRODUCER_PROPERTIES_SASL_JAAS_CONFIG: org.apache.kafka.common.security.plain.PlainLoginModule required username="Q4TARSDCREBSDSDGRE" password="Fscjer00sBDLwbrPSOPSmQdKLT0psSDTSDfRI6pgrAxfABD09aQzDdre2SDrfesd";
  EVENTUATE_LOCAL_KAFKA_PRODUCER_PROPERTIES_SECURITY_PROTOCOL: SASL_SSL
  EVENTUATE_LOCAL_KAFKA_PRODUCER_PROPERTIES_BOOTSTRAP_SERVERS: pkc-6ksd5.us-east1.gcp.confluent.cloud:9092

**Comment by cer** [comment]
@dartartem Does the CDC now support the configuration described in comment https://github.com/eventuate-foundation/eventuate-cdc/issues/60#issuecomment-654468516

**Comment by dartartem** [comment]
@cer, yes, it supports generic properties, but prefix is different: **EVENTUATE_LOCAL_KAFKA_CONSUMER**_SASL_MECHANISM

**Comment by cer** [comment]
What about producer properties?

**Comment by dartartem** [comment]
@cer, kafka healthcheck does not use producer. But cdc producer already supports generic properties: EVENTUATE_LOCAL_KAFKA_**PRODUCER**_SASL_MECHANISM: PLAIN
